waspc/ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.10.6
+
+### Bug fixes
+- Wrap react routes with `<Switch/>` so that only one route is rendered at a time.
```

Comment:
The wording is perhaps a bit too low-level and unclear for end users. How about:
```suggestion
- Wasp now renders only the first route that matches the current path in the browser. 
```